docs/section19/19-19.md

```diff
@@ -0,0 +1,94 @@
+                - Get the enthalpy recovery ratio: `enthalpy_rec_ratio = fan_sys_b.air_energy_recovery.enthalpy_recovery_ratio`  
+                - Assume that the outdoor air flow through the ERV is equal to the minimum outdoor airflow: `ERV_OA_air_flow = fan_sys_b.minimum_outdoor_airflow`  
+                - Assume that the exhaust air flow through the ERV is equal to the maximum of either the total exhaust cfm or total relief cfm associated with the fan system: `ERV_Ex_air_flow = max(total_exhaust_cfm,total_relief_cfm)`  
+                - Calculate A per Section 6.5.3.1.1: `A = (((2.2*enthalpy_rec_ratio)-0.5) *(ERV_Ex_air_flow + ERV_OA_air_flow))/4131`  
+            
+            - Check if system type is constant volume:  `If any(hvac.id in baseline_hvac_system_dict[sys_type] for sys_type in ["SYS-3", "SYS-3a", "SYS-3b", "SYS-3c", "SYS-4", "SYS-12", "SYS-12a", "SYS-12b", "SYS-12c","SYS-13", "SYS-13a"]): `   
+                - Calculate the expected BHP (if no other pressure drop adjustments apply): `expected_BHP = (0.00094 * supply_cfm_b) + A`  
+            - Else if the system type is variable volume and not system 11: `Elif any(hvac.id in baseline_hvac_system_dict[sys_type] for sys_type in ["SYS-5", "SYS-5b", "SYS-6", "SYS-6b","SYS-7", "SYS-7a", "SYS-7b", "SYS-7c","SYS-8", "SYS-8a", "SYS-8b", "SYS-8c"]): `  
+                - Calculate the expected BHP (if no other pressure drop adjustments apply): `expected_BHP = (0.0013 * supply_cfm_b) + A`  
+            - Else, the system is system 11: `Else:`  
+                - Calculate the expected BHP (if no other pressure drop adjustments apply): `expected_BHP = (0.00062 * supply_cfm_b) + A`  
+
+            - Get the motor efficiency from Table G3.9.1 (should be looked up based on the next motor size greater than the bhp (i.e., expected_BHP)): `motor_eff = data_lookup(G3.9.1,expected_BHP)`  
+            - Calculate the expected Wattage (assuming no other pressure drop adjustments apply except energy recovery): `fan_Wattage_expected = expected_BHP * 746 *(1/motor_eff)`  
+
+            - **Rule Assertion:** 
+            - Case 1: If fan wattage expected equals total fan power: `if total_fan_power == fan_Wattage_expected: outcome = "PASS"`  
+            - Case 2: Elif conservative comparison less equal:`elif AHJ_RA_compare == True and total_fan_power <= fan_Wattage_expected: PASS`
+            - Case 3: Else, undetermined: `Else: outcome = "UNDETERMINED" and raise_message "Fan power for <hvac.id> is greater than expected per Section and Table G3.1.2.9 assuming no pressure drop adjustments per Table 6.5.3.1-2 other than energy recovery (if modeled). Conduct manual check."`  
+
+**Notes/Questions:**  
+1. I don't need case 1 but I put it here just so the RCT could see the "perfect" result.  
+2. There is not an explicit schema data element for ERV exhaust and OA air flow so I made some assumptions. See above.
```

Comment:
Let's discussion on the RDS call to make sure that others agree with the assumptions. They look good to me.